service/src/util.rs

```diff
@@ -43,10 +43,15 @@ pub fn package_version() -> Result<PackageVersion, IndexerError> {
             .as_str()
             .unwrap()
             .to_string();
-        let dependencies = pkg.get("dependencies").and_then(|d| d.as_table()).unwrap();
-        let indexer_native = dependencies
-            .get("indexer-native")
-            .map(|d| d.as_str().unwrap().to_string());
+        let dependencies = pkg
+            .get("dependencies")
+            .and_then(|d| d.as_table())
+            .expect("Parse package dependencies");
+        let indexer_native = dependencies.get("indexer-native").map(|d| {
+            d.as_str()
+                .expect("Parse indexer-service dependency version")
+                .to_string()
+        });
```

Comment:
Since this is only called once on startup, I would just make any errors here fatal, i.e. use expect and no `IndexerError`. Only errors while running are worth tracking in my opinion.

Reply:
I'm a bit confused 🧐 I didn't add `IndexerError` here, the only changes were from `unwrap` to `expect` for a better error message when it fails fatally

Reply:
I kept the `expect` messages changes and removed `IndexerError` from helper fn `read_manifest`

Reply:
> I'm a bit confused 🧐 I didn't add `IndexerError` here, the only changes were from `unwrap` to `expect` for a better error message when it fails fatally

Ah, sorry. I sometimes comment on things even if they were not introduced in the PR itself. Sometimes, a PR is a good opportunity to improve code that was there before.